Firestore/core/src/firebase/firestore/util/filesystem.h

```diff
@@ -72,6 +72,12 @@ Status RecursivelyDelete(const Path& path);
  */
 Path TempDir();
 
+/**
+ * On success, sets `size` to be the size in bytes of the file specified by
+ * `path`.
+ */
+Status GetFileSize(const Path& path, off_t* size);
```

Comment:
How about `StatusOr<off_t>` as a return type?

`TempDir` similarly returns a value but isn't "GetTempDir". Could this drop the "Get" and be `FileSize`?

Reply:
Done on both counts.